deployment/sids-survey-backend-deployment.yml

```diff
@@ -0,0 +1,86 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: sids-survey-backend
+  namespace: ml-app
+spec:
+  selector:
+    app: sids-survey-backend
+  ports:
+  - protocol: TCP
+    port: 80
+    targetPort: 80
+  type: ClusterIP
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sids-survey-backend
+  namespace: ml-app
+  labels:
+    app: sids-survey-backend
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: sids-survey-backend
+  template:
+    metadata:
+      labels:
+        app: sids-survey-backend
+    spec:
+      containers:
+      - name: sids-survey-backend
+        image: acrmlbackend.azurecr.io/sidmlbackend:latest
+        resources:
+          limits:
+            cpu: 2000m
+            memory: 2048Mi
+          requests:
+            cpu: 100m
+            memory: 512Mi
+        volumeMounts:
```

Comment:
If you don't have a shared dataset or no need of shared file storage can delete volumeMount as well